test/functional/page_objects/visualize_page.js

```diff
@@ -248,17 +248,28 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
 
     async setComboBox(comboBoxSelector, value) {
       const comboBox = await testSubjects.find(comboBoxSelector);
-      await this.setComboBoxElement(comboBox, value);
+      await this.setComboBoxValue(comboBox, value);
+      await this.closeComboBoxOptionsList(comboBoxSelector);
       await PageObjects.common.sleep(1000);
```

Comment:
wonder if we could get rid of this sleep as part of this code?

Reply:
I'll try it out.